src/Components/Modal/Modal.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import './Modal.css';
+import { NavLink } from 'react-router-dom/cjs/react-router-dom.min';
+import { ReactComponent as CloseIcon } from '../../icons/X.svg';
+
+const Modal = (props) => {
+  const closeModal = () => {
+    props.setIsModalOpen(false);
+  };
+
+  return (
+    <nav className="modal-nav">
+      <CloseIcon
+        className="close-icon"
+        alt="Close menu"
+        onClick={closeModal}
+        onKeyDown={(event) => {
```

Comment:
Needed for accessibility. Without this code, if someone navigated to the close icon the modal would immediately close. This code checks to make sure that they've hit the enter key, which is typical for buttons, and then invokes `closeModal` if that's the case.